test/integration/search/search_test.rb

```diff
@@ -102,7 +102,7 @@ def test_filter_by_field
 
     get "/search?filter_mainstream_browse_pages=browse/page/1"
 
-    assert_equal ["/government-1", "/mainstream-1"],
+    assert_equal ["/government-1", "/govuk-1", "/mainstream-1"],
```

Comment:
Would like to discuss this as we would want to exclude any data from the `govuk` index from the results for the time being. Is this here intentionally or because it was the easiest way to set everything up?